### DISCUSSION TITLE
Add more icons

### DISCUSSION THREAD

**Discussion by vishal-quadralyst** [twbs/bootstrap#38945] [Ideas]
Sir/ Madam, we need more icon related with order status, issue track status ( like -  open, in progress, waiting, resolved, close, re-open) and many more because we are using only bootstrap icon

**Comment by julien-deramond** [comment]
Hi @vishal-quadralyst 
You can request icons by [creating an icon request issue](https://github.com/twbs/icons/issues/new?assignees=&labels=icon-request&projects=&template=icon-request.md&title=) so it ends up in the [twbs/icons backlog](https://github.com/twbs/icons/issues).
